### ISSUE TITLE
Dedupe more libraries

### ISSUE THREAD

**Issue by KyleMaas** [arj03/ssb-browser-demo#240]
Other libraries we use multiple copies of:

* chloride 2.2.14 pulled in by ssb-keys-mnemonic vs. 2.4.1 everywhere else
* ssb-keys 7.2.2 almost everywhere except ssb-db2's 8.0.2 (ssb-room/ssb-client, ssb-room/ssb-config, ssb-validate, ssb-client, ssb-client/ssb-config)
* ssb-config 3.3.3 from ssb-room and ssb-room/ssb-client vs 3.4.5 from ssb-client (direct dependency)
* sodium-native 2.4.9 (which doesn't work for me) pulled in by ssb-keys-mnemonic/chloride vs. 3.2.1 used by ssb-browser-core/ssb-keys/chloride

I'd keep going, but I'm finding a lot more.  Let me see what I can do to fix some of these.  Some of them are minor version changes that shouldn't make a lick of difference.

Filed issues on this subject:

* https://github.com/staltz/ssb-keys-mnemonic/issues/2
* https://github.com/flumedb/flumeview-reduce/issues/17
* https://github.com/flumedb/flumecodec/issues/2
* https://github.com/push-stream/push-stream-to-pull-stream/issues/2
* https://github.com/dweinstein/pull-async-filter/issues/1
* https://github.com/dominictarr/pull-goodbye/issues/2
* https://github.com/staltz/ssb-room/issues/28
* https://github.com/staltz/ssb-conn-db/issues/3
* https://github.com/dominictarr/sodium-browserify/issues/7

**Comment by arj03** [comment]
I think if you create PRs like for the ssb-keys-mnemonic we should be able to get them merged. The pull* modules might be a bigger problem as they are not actively maintained sadly. The flume stuff shoul not be used. Might be pulled in like in friends but shouldn't be used.

**Comment by KyleMaas** [comment]
flumecodec is pulled in by ssb-friends (via flumeview-reduce) and atomic-file, both dependencies of ssb-browser-core.

**Comment by arj03** [comment]
Right, this reminds me, we need to remove the atomic-file dependency in ssb-browser-core in favor of atomically-universal. Is that something you are up for?

**Comment by arj03** [comment]
https://github.com/arj03/ssb-browser-core/issues/10

**Comment by KyleMaas** [comment]
https://github.com/staltz/ssb-keys-mnemonic/pull/3

**Comment by KyleMaas** [comment]
https://github.com/ssbc/ssb-friends/issues/56

**Comment by KyleMaas** [comment]
https://github.com/arj03/ssb-browser-core/pull/29

**Comment by KyleMaas** [comment]
https://github.com/dominictarr/sodium-browserify-tweetnacl/issues/3

https://github.com/dominictarr/sodium-browserify-tweetnacl/issues/4

**Comment by KyleMaas** [comment]
See #244 for a patch for some of these.

**Comment by KyleMaas** [comment]
I think I've done about everything I can to fix this locally.  I'm going to leave this open to show progress on dependencies, but for now, I don't think there's much more we can do here.
